channel-diagrams.tex

```diff
@@ -437,7 +437,7 @@
 }
 
 \tikzset{
-  circAB/.style={ draw, circle, fill halves={blue!50,red!50}, minimum size=4mm }
+  circAB/.style={ draw, circle, fill halves={red!50,blue!50}, minimum size=4mm }
```

Comment:
This change is debatable: the split circle is used throughout the paper, including places where it allocated to Bob then Alice, or just to Bob. So the colour-coding does not denote anything about the outcome of that channel, only the participants of that channel. 

That said I still prefer it this way. The participants are also _ordered_ A,B so the colouring should be, too.

Reply:
It affects Figures 3,5,6,7,8,9,10,13,14,15,16,17,18,19,20,24,25,26